packages/adblocker-extended-selectors/src/types.ts

```diff
@@ -35,7 +35,7 @@ export type PseudoClass = Base & {
   type: 'pseudo-class';
   name: string;
   argument: string | undefined;
-  subtree: AST | undefined;
+  subtree?: AST | undefined;
```

Comment:
```suggestion
  subtree?: AST;
```

Also, `argument?: string;` above (for consistency)

Reply:
reverted to old code - this one should never have been committed